ExpressWash/ExpressWash/AddCarViewController.swift

```diff
@@ -61,23 +63,36 @@ UIImagePickerControllerDelegate, UITextFieldDelegate {
     }
 
     private func setupCamera() {
-        let camera = UIImagePickerController()
-        camera.sourceType = .camera
-        camera.allowsEditing = true
-        camera.delegate = self
-        present(camera, animated: true)
+        if AVCaptureDevice.authorizationStatus(for: .video) ==  .authorized {
+            carImagePicker.delegate = self
+            carImagePicker.sourceType = .camera
```

Comment:
Please add this check so the simulator doesn't crash when you run this. Thanks!
```suggestion
            // try to access the camera
                    if UIImagePickerController.isSourceTypeAvailable(.camera) {
                        self.imagePicker.sourceType = .camera
                    } else {
                        // if camera not available (simulator) access saved photos
                        self.imagePicker.sourceType = .savedPhotosAlbum
                    }
```

Reply:
See commit below.